sdk/keyvault/test-resources-cleanup.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+from azure.identity import ClientSecretCredential
+from azure.keyvault.certificates import CertificateClient
+from azure.keyvault.keys import KeyClient
+from azure.keyvault.secrets import SecretClient
+
+if "AZURE_KEYVAULT_URL" not in os.environ:
+    raise EnvironmentError("Missing a Key Vault URL")
+if "KEYVAULT_TENANT_ID" not in os.environ:
+    raise EnvironmentError("Missing a tenant ID for Key Vault")
+if "KEYVAULT_CLIENT_ID" not in os.environ:
+    raise EnvironmentError("Missing a client ID for Key Vault")
+if "KEYVAULT_CLIENT_SECRET" not in os.environ:
+    raise EnvironmentError("Missing a client secret for Key Vault")
+
+credential = ClientSecretCredential(
+    tenant_id=os.environ.get("KEYVAULT_TENANT_ID"),
+    client_id=os.environ.get("KEYVAULT_CLIENT_ID"),
+    client_secret=os.environ.get("KEYVAULT_CLIENT_SECRET")
+)
+
+if "AZURE_KEYVAULT_URL" in os.environ:
```

Comment:
This will always be True (if it were False, we would have raised on line 9)

Reply:
Thanks for the catch -- that was a remnant of when I had prototyped this to handle MHSM cleanup as well, but I decided that was best left for another time